.github/workflows/build.yml

```diff
@@ -63,13 +74,15 @@ jobs:
 
       # Zip dist folder
       - name: Zip build output
+        if: github.event_name == 'pull_request' && inputs.deploy == true
         shell: bash
         run: |
             cd ${{ inputs.package-dist-directory }}
             zip -r ./${{ env.ZIP_NAME }} .
 
       # Upload zip to S3
       - name: Upload to S3
+        if: github.event_name == 'pull_request' && inputs.deploy == true
```

Comment:
Out of curiosity, does S3 automatically unzips the file?

Reply:
@fernandofranca I think it's Amplify that handles the zip

Reply:
@fernandofranca - Yep, @xander-marjoram is correct. When you upload files from S3 to Amplify, they need to be in zip format :)